docs/app/pages/Components/Chips/examples/Format.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <md-chips class="md-primary" v-model="clubs" md-placeholder="Add club..." :md-format="toUppercase">
+      <label>La Liga Clubs</label>
+      <div class="md-helper-text">Three uppercase letters</div>
+    </md-chips>
+    <md-chips class="md-primary" v-model="artists" md-placeholder="Add artist..." :md-format="formatName">
+      <label>Artists</label>
+      <div class="md-helper-text">Try insert `Eugène Ysaÿe` twice. The formatter will remove diacritics.</div>
+    </md-chips>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'DuplicatedFeedback',
+  data: () => ({
+    clubs: [
+      'FCB',
+      'MAD'
+    ],
+    artists: [
+      'Claude Debussy',
+      'Jules Massenet',
+      'Gabriel Dupont',
+      'Emma Bardac',
+      'Mary Garden'
+    ]
+  }),
+  methods: {
+    toUppercase (str) {
+      str = str.replace(/\s/g, '').toUpperCase()
+      if (str.length !== 3) return false
+      return str
+    },
+    formatName (str) {
+      let words = str.split(' ').filter(str => str !== '')
+      // remove accents / diacritics
+      words = words.map(str => str.normalize('NFD').replace(/[\u0300-\u036f]/g, ''))
+      // capitalize
+      words = words.map(str => str[0].toUpperCase() + str.slice(1))
+      let result = words.join(' ')
+      return result
```

Comment:
Just return the `words.join(' ')` instead of creating a new var

Reply:
fix e955e997a01d259c6bef81617e26ea98b85d9b4c